Docs changes for platform (#329)
hirosystems/docs: docs/platform/overview.md

---
title: Overview
---

The [Hiro Platform](https://platform.hiro.so/) is a cloud-based development environment that enables you to create, build and deploy a Clarity smart contract. The Hiro platform already contains the developer tools and resources to develop smart contracts.

## Hiro Platform design and architecture

We want to encourage developers to focus on shipping their business models to Stacks blockchain, ship often and happily, and avoid getting caught up with unboxing and infrastructure overhead.

To deliver that developer productivity, the Hiro Platform comes pre-installed with the following developer tools and resources to help speed up the developer journey:

> - Web-based VS Code

Reviewer: ```suggestion
- VS Code in Web Browser
```

---

> title: Hiro Platform Overview

Reviewer: ```suggestion
title: Overview
```

---

## Introduction

> The [Hiro Platform](https://platform.hiro.so/) is a cloud-based "sandbox" environment that enables you create, build and deploy a contract in less than 5 minutes. The Hiro platform already contains the tools and resources you need to create smart contracts.

Reviewer: ```suggestion
The [Hiro Platform](https://platform.hiro.so/) is a cloud-based development environment that enables you to create, build and deploy a Clarity smart contract in less than 5 minutes. The Hiro platform already contains the developer tools and resources to develop smart contracts.
```

## Hiro Platform design and architecture

> The Hiro Platform was designed to make it easy to deploy contracts in a secure, cloud-based testing environment without placing many requirements on how you choose to use the Platform. When granted access to the Platform (via the sign-in process), only you have access to your own testing environment. No limitations or restrictions are placed on how many contracts you can work with or deploy, and Hiro will not require you to perform any additional installation or configuration steps to deploy contracts.

Reviewer: ```suggestion
We want to encourage developers to focus on shipping their business models to Stacks blockchain, ship often and happily, and avoid getting caught up with unboxing and infrastructure overhead.
```

> The Hiro Platform comes pre-installed with the following developer tools and resources to help speed up the deployment process:

Reviewer: ```suggestion
To deliver that developer productivity, the Hiro Platform comes pre-installed with the following developer tools and resources to help speed up the developer journey:
```

> - VSCode

Reviewer: ```suggestion
- Web-based VS Code
- Clarinet
- Clarity for VS Code
- Git
```

> - Git

Reviewer: ```suggestion
```

> - Clarity

Reviewer: ```suggestion
```

---
title: Overview
---

The [Hiro Platform](https://platform.hiro.so/) is a cloud-based development environment that enables you to create, build and deploy Clarity smart contracts. The Hiro platform has pre-installed developer tools and helpful resources to improve developer productivity.

## Purpose

We want to encourage developers to focus on shipping their business models to Stacks blockchain, ship often and happily, and avoid getting caught up with unboxing and infrastructure overhead.

To deliver developer productivity, the Hiro Platform comes pre-installed with the following developer tools and resources to help speed up the developer journey:

> - Example Projects

Reviewer: Let the new order be:

- VS Code in Web Browser
- Clarinet
- Clarity for VS Code
- Clarity Examples

- VS Code in Web Browser
- Clarinet
- Clarity for VS Code
- Git

These tools enable you to quickly get up and running with the Platform without requiring additional steps. Less setup time equals more time for translating your business models into code, testing them, and deploying them to a Stacks network!

:::note

Please note that this is a BETA version of the Hiro Platform, and the product is undergoing stability improvements and testing before the general availability release.

> While using the BETA release of the Hiro Platform, Hiro will not be liable for any data loss as described in the [Terms of Service](http://hiro.so/terms).

Reviewer: ```suggestion

While using the BETA release of the Hiro Platform, Hiro will not be liable for any data loss as described in the [Terms of Service](http://hiro.so/terms).
```

> Let us know at platform@hiro.so if you encounter any bugs or availability issues.

Reviewer: ```suggestion

Let us know at platform@hiro.so if you encounter any bugs or availability issues.
```

Reviewer: Wanted to add a line-break between.
